components/order-app/checkout/stripe-utils.ts

```diff
@@ -50,6 +53,7 @@ export default async function handleCheckoutRequest(showSpinner, showError) {
     }
     const stripe = await loadStripe('pk_live_ivfkFrzhLuZbUiZRVkvsBwI3');
     // const stripe = await loadStripe('pk_test_OaDvLsgEGQbshVWpSFMQMm1k');
+    if (!stripe) return null;
```

Comment:
If this is necessary for type checking, think it should throw an error instead

Reply:
Should I write this?

```ts
```

Reply:
Yes